### PR TITLE
 init CVE-2019-10135

### PR DESCRIPTION
**Pull request by rehakm**

### CODE REVIEW

**Comment by iamleot** [comment]
Hello rehakm, 
can you please add a reference to the corresponding upstream pull request?:

 <https://github.com/containerbuildsystem/osbs-client/pull/865>

...and clarify a bit the version affected changing the description from:

 `versions 0.46.x to 0.56.1'

to:

 `versions since 0.46 before 0.56.1'

(seems fixed in 0.56.1)


Thank you!

File: 2019/10xxx/CVE-2019-10135.json [hunk 1]
```diff
@@ -4,15 +4,73 @@
     "data_version": "4.0",
```
Comment (cve-team) [inline 2019/10xxx/CVE-2019-10135.json]: Invalid source.

Comment (cve-team) [inline 2019/10xxx/CVE-2019-10135.json]: MISC or CONFIRM, but name != url for reference.

**Comment by cve-team** [comment]
Thank you for your submission, expect CVE-2019-10135 to be published on http://cve.mitre.org in the next few hours.

**Pull request merged**